packages/dev/gui/src/2D/controls/rectangle.ts

```diff
@@ -57,7 +96,7 @@ export class Rectangle extends Container {
 
     /** @internal */
     protected _computeAdditionnalOffsetX() {
-        if (this._cornerRadius) {
+        if (this._cornerRadiusFree) {

```

Comment:
this will always return true

Reply:
Yup if we're doing the check in a Vector4/array of numbers, we have to check if any member is !== 0.